waspc/examples/todo-typescript/tsconfig.json

```diff
@@ -16,11 +16,12 @@
       "esnext"
     ],
     "allowJs": true,
+    "types": ["@testing-library/jest-dom"],
```

Comment:
TS complains about `toBeInTheDocument` and other matchers to be undefined if we don't include this. 

This means that the user `tsconfig.json` needs to include this for the type support to work properly.

Reply:
Can we do this differently somehow? @sodic

Reply:
Hmm, how did it work before restructuring?

I know [this](https://www.typescriptlang.org/tsconfig#typeRoots) works, but I remember reading a TS issue recently where people complained about TS not looking for types in directories outside of `@types`. I'll try to find it and see what they decided.

Edit: It was [this one](https://github.com/microsoft/TypeScript/issues/36042), kind of related.